resources/build/build-utils.sh

```diff
@@ -390,29 +390,29 @@ function _builder_failure_trap() {
       target=:project
     fi
 
-    builder_report failure $action $target
+    builder_finish_action failure $action $target
   done
 
   _builder_current_actions=()
 }
 
 #
-# Builds the standarized `action:target` string for the specified action-target pairing
-# and also returns 0 if the user has asked to perform it on the command line.  Otherwise,
-# returns 0 and sets an empty string in place of the matched pair.
+# Builds the standarized `action:target` string for the specified action-target
```

Comment:
```suggestion
# Builds the standardized `action:target` string for the specified action-target
```

I made the typo in the base branch, to be sure, but in order to prevent merge conflict issues, may as well correct it here instead.